src/api/requests/Websocket/Websocket.ts

```diff
@@ -1,45 +1,50 @@
-import {Centrifuge, MessageContext} from "centrifuge";
-import {BaseWebsocketModel, WebsocketTransaction} from '@/api/types/transaction'
+import { Centrifuge, MessageContext } from 'centrifuge'
+import { BaseWebsocketModel, WebsocketTransaction } from '@/api/types/transaction'
+import { toast } from 'react-toastify'
 
 export const SetupWebsocket = (url: string) => {
-    let retries = 0;
-    const centrifuge = new Centrifuge(url, {websocket: WebSocket});
-    centrifuge.connect();
-
-    centrifuge.on('message', function (ctx: MessageContext) {
-        const res = parseResponse(ctx)
-
-        if (typeof res == "object" && res != undefined){
-            switch (res.eventType) {
-                case "create_transaction":
-                    // TODO: create notification
-                    break;
-            }
-        }
-    });
-
-    centrifuge.on('error', function () {
-        retries++
-        if (retries >= 3){
-            centrifuge.disconnect()
-        }
-    })
-
-    const channel = "bux-wallet";
-    const sub = centrifuge.newSubscription(channel);
-    sub.subscribe();
+  let retries = 0
+  const centrifuge = new Centrifuge(url, { websocket: WebSocket })
+  centrifuge.connect()
+
+  centrifuge.on('message', function (ctx: MessageContext) {
+    const res = parseResponse(ctx)
```

Comment:
### minor
I feel like "response" is not the best word describing message that comes from the server.
Although currently it contains only messages that relates to creating transaction, hopefully we will extend bux-server functionality to push some notification to wallets, so bux-wallet-backend will be able to send more informations here.

Reply:
Renamed to event